projects/canopy/src/lib/docs/principles/accessibility/developer.stories.mdx

```diff
@@ -0,0 +1,225 @@
+import { Meta } from '@storybook/addon-docs';
+import Feedback from '../../common/feedback.md';
+
+<Meta title="Principles/Accessibility/Developer" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# For developers
+
+<p class="standfirst">Here we provide information on developing content to be accessible.</p>
+
+## Developing with accessibility in mind
+
+A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times.
+
+WAI-ARIA, the Accessible Rich Internet Applications Suite, can be used to make web content and web applications more accessible to people with disabilities.
+
+WAI-ARIA is especially helpful when creating dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.
+
+
+**Components from the Canopy Angular library have been built to be accessible.**
+
+### Defining the language of the document
+
+Defining the language used in an HTML page has many benefits. It explicitly tells the browser what language you are using, thus improving SEO. Third-party translation tools and browsers can use the language information to use the correct dictionary. Additionally, assistive technologies can choose the suitable voice profile or character set.
+
+```html
+<html lang="en">
+  ...
+</html>
+```
+
+If you switch language within a document you can use the lang attribute on specific tags:
+
+```html
+<p>You could have <i lang="it">una frase in italiano<i></p>
+```
+
+### Provide a "skip to main content" link
+
+When keyboard-only users interact with your site they use the tab key to jump from link to link. If you have a lot of links in your header or in a menu, they must tab through those every time they come to a new page just to get to the main content. Providing a skip to main content link allows them to easily bypass this.
```

Comment:
Just a suggestion:

When keyboard-only users interact with your site, they use the tab key to jump from link to link. 

Having a lot of link elements in your header or menu causes users to unnecessarily tab through each link before they can get to the main content.

Providing a link that allows users to bypass the header and skip to the main content improves accessibility.

You can do that by using the `skip-content` CSS class: